console/client/src/features/modules/ModuleDetails.tsx

```diff
@@ -32,9 +41,12 @@ export function ModuleDetails() {
       }
       setTabs([ ...tabs, newTab ])
     }
-    setActiveTab(existingTab ? index : tabs.length)
-    searchParams.set(TabSearchParams.active, newTab?.id ?? tabs[index].id)
-    navigate({ ...location, search: searchParams.toString() })
+    setActiveTab({ id: tabId, type: TabType.Verb })
+    setSearchParams({
+      ...Object. fromEntries(searchParams),
```

Comment:
Is there an extra space here? `...Object. fromEntries(searchParams),` seems like this should be `...Object.fromEntries(searchParams),`

Maybe the linter is missing a rule for this?